webpack/raygun-webpack-plugin/src/raygun-webpack-plugin.ts

```diff
@@ -2,16 +2,38 @@ import * as https from 'https';
 import { Compiler } from 'webpack';
 
 interface RaygunWebpackPluginOptions {
+    // Required options
     applicationId: string;
     patToken: string;
     baseUri: string;
+    // Optional rate limiting configuration
+    retryConfig?: {
+        maxRetries?: number;        // Maximum number of retry attempts (default: 3)
+        initialDelay?: number;      // Initial delay in ms before first retry (default: 1000)
+        maxDelay?: number;          // Maximum delay in ms between retries (default: 30000)
+        backoffFactor?: number;     // Multiplier for exponential backoff (default: 2)
+    };
+    delayBetweenFiles?: number;    // Optional: Add delay between file uploads if you're hitting rate limits
```

Comment:
```suggestion
    delayBetweenFiles?: number;    // Add delay between file uploads if you're hitting rate limits
```

Since it's all optional